python/python/glide/async_commands/server_modules/json.py

```diff
@@ -193,6 +193,61 @@ async def arrlen(
     )
 
 
+async def arrtrim(
+    client: TGlideClient,
+    key: TEncodable,
+    path: TEncodable,
+    start: int,
+    end: int,
+) -> TJsonResponse[int]:
+    """
+    Trims an array at the specified `path` within the JSON document stored at `key` so that it becomes a subarray [start, end], both inclusive.\n
+    If `start` < 0, it is treated as 0.\n
+    If `end` >= size (size of the array), it is treated as size-1.\n
+    If `start` >= size or `start` > `end`, the array is emptied and 0 is returned.\n
```

Comment:
Don't include `\n`s

Reply:
it is more readable in the IDE
![image](https://github.com/user-attachments/assets/fdb1ab6b-bf03-4d3f-b21a-a94c779d42fc)


no \n:

![image](https://github.com/user-attachments/assets/5d58ae1a-2f30-4643-85b9-9c5d841f0ea2)

Reply:
let me know what you thing, removed for now